compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -1043,6 +1043,15 @@ impl<'hir> Map<'hir> {
             _ => None,
         }
     }
+
+    pub fn impl_parent(self, did: LocalDefId) -> Option<LocalDefId> {
```

Comment:
Should this get another name to avoid the confusion with the `impl_parent` query? Or better: rename the `impl_parent` query?

Reply:
we probably don't need it, I was just misunderstanding impl_parent and was curious why there was no local provider, only a provider for external crates.